### PR TITLE
chore: roll to include setChecked

### PR DESCRIPTION
**Pull request by avodovnik**

### CODE REVIEW

File: src/Playwright/API/Generated/.editorconfig [hunk 1]
```diff
@@ -15,4 +15,8 @@ dotnet_diagnostic.SA1606.severity = none
 dotnet_diagnostic.SA1614.severity = none
 dotnet_diagnostic.SA1615.severity = none
 dotnet_diagnostic.SA1618.severity = none
-dotnet_diagnostic.SA1629.severity = none
\ No newline at end of file
+dotnet_diagnostic.SA1629.severity = none
+dotnet_diagnostic.CA1041.severity = none # [Obsolete] can exist without message
+dotnet_diagnostic.CS0612.severity = none # We can use [Obsolete] in generated code
+dotnet_diagnostic.SA1300.severity = none # Sometimes, we want to use _ in the name
```
Comment (mxschmitt) [inline src/Playwright/API/Generated/.editorconfig:21]: when do we want to use _ in the name?

File: src/Playwright.Tests/BrowserTypeConnectTests.cs [hunk 2]
```diff
@@ -50,7 +50,7 @@ public void SetUpAsync()
                     StartInfo =
                     {
                         FileName = GetExecutablePath(),
-                        Arguments = $"launch-server {BrowserType.Name}",
+                        Arguments = $@"launch-server {BrowserType.Name}",
```
Comment (mxschmitt) [inline src/Playwright.Tests/BrowserTypeConnectTests.cs:53]:
```suggestion
                        Arguments = $"launch-server {BrowserType.Name}",
```

File: src/Playwright.Tests/PageEventNetworkTests.cs [hunk 3]
```diff
@@ -130,14 +130,14 @@ public async Task ShouldSupportRedirects()
             string FOO_URL = Server.Prefix + "/foo.html";
             var response = await Page.GotoAsync(FOO_URL);
             await response.FinishedAsync();
-            Assert.AreEqual(new[] {
+            CollectionAssert.AreEquivalent(new[] {
```
Comment (mxschmitt) [inline src/Playwright.Tests/PageEventNetworkTests.cs:133]: what is there the difference to Assert.AreEqual?

File: src/Playwright/API/Generated/.editorconfig (hunk 1 above)
Comment (mxschmitt) [reply src/Playwright/API/Generated/.editorconfig:21]: like in the other PR, please revert.

File: src/Playwright/Transport/Converters/EvaluateArgumentValueConverter.cs [hunk 4]
```diff
@@ -260,18 +260,25 @@ private static object ParseEvaluateResult(JsonElement result, Type t, JsonSerial
...
 
-                return objResult;
+                        var property = t.GetProperties().FirstOrDefault(prop => string.Equals(prop.Name, kv.K, StringComparison.OrdinalIgnoreCase));
+                        serializerOptions.Converters.Add(GetNewConverter(property.PropertyType));
+                        property.SetValue(objResult, kv.V.ToObject(property.PropertyType, serializerOptions));
+                    }
+
+                    return objResult;
+                }
+                catch (Exception)
```
Comment (mxschmitt) [inline src/Playwright/Transport/Converters/EvaluateArgumentValueConverter.cs:278]: is this change related to the roll?

Comment (avodovnik) [reply src/Playwright/Transport/Converters/EvaluateArgumentValueConverter.cs:278]: Sort of. It's a consequence of a problem I was diagnosing, but it also makes sense to do it parallel to this.

File: src/Playwright.Tests/PageEventNetworkTests.cs (hunk 3 above)
Comment (avodovnik) [reply src/Playwright.Tests/PageEventNetworkTests.cs:133]: `AreEqual` assumes order is the same, while `AreEquivalent` ensures all items are there, but can be in different orders.

File: src/Playwright.Tests/BrowserTypeConnectTests.cs (hunk 2 above)
Comment (avodovnik) [reply src/Playwright.Tests/BrowserTypeConnectTests.cs:53]: This was likely a conflict merge gone the wrong way -.-

File: src/Playwright.Tests/PageEventNetworkTests.cs (hunk 3 above)
Comment (mxschmitt) [reply src/Playwright.Tests/PageEventNetworkTests.cs:133]: We want to have AreEqual in most cases, otherwise we use Sets upstream.

File: src/Playwright.Tests/PageNetworkRequestTest.cs [hunk 5]
```diff
@@ -235,5 +235,78 @@ public async Task ShouldReturnNavigationBitWhenNavigatingToImage()
...
+            Page.Request += async (sender, r) =>
+            {
+                await (await r.ResponseAsync()).FinishedAsync();
+                tsc.SetResult(r.Sizes);
+            };
+            await Page.EvaluateAsync("() => fetch('./get').then(r => r.text())");
+
+            var sizes = await tsc.Task;
+            Assert.AreEqual(0, sizes.RequestBodySize);
+            Assert.GreaterOrEqual(sizes.RequestHeadersSize, 228);
```
Comment (mxschmitt) [inline src/Playwright.Tests/PageNetworkRequestTest.cs:270]:
```suggestion
            Assert.GreaterOrEqual(sizes.RequestHeadersSize, 200);
```

File: src/Playwright.Tests/PageNetworkRequestTest.cs [hunk 6]
```diff
@@ -235,5 +235,78 @@ public async Task ShouldReturnNavigationBitWhenNavigatingToImage()
...
+            Page.Request += async (sender, r) =>
+            {
+                await (await r.ResponseAsync()).FinishedAsync();
+                tsc.SetResult(r.Sizes);
+            };
+            await Page.EvaluateAsync("() => fetch('./get', { method: 'POST', body: '12345'}).then(r => r.text())");
+
+            var sizes = await tsc.Task;
+            Assert.AreEqual(5, sizes.RequestBodySize);
+            Assert.GreaterOrEqual(sizes.RequestHeadersSize, 300);
```
Comment (mxschmitt) [inline src/Playwright.Tests/PageNetworkRequestTest.cs:253]:
```suggestion
            Assert.GreaterOrEqual(sizes.RequestHeadersSize, 200);
```

**Review by mxschmitt — APPROVED** [review]
